### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by pclausen**
Adding the hello world example with cmake and ctest

### CODE REVIEW

File: exercises/hello-world/hello_world_test.f90 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+! This test was created from ..\..\exercism\problem-specifications\exercises\hello-world\canonical-data.json
+! version: 1.1.0 
+!
+program hello_world_test_main
+  use TesterMain
+  use hello_world 
+
+  implicit none
+
+  !! About the test comments:
```
Comment (kotp) [inline exercises/hello-world/hello_world_test.f90]: Need to expand on this a little bit, as it is the first time students will see this.  Something small, perhaps even referring back to the "README" file, perhaps mentioning the "Instructions" tab on the web.

**Comment by kotp** [comment]
This relies on #42 to be brought in, as it uses the new `test_report` name that replaces the old `test_end`.

File: exercises/hello-world/hello_world_test.f90 (hunk 1 above)
Comment (pclausen) [reply exercises/hello-world/hello_world_test.f90]: I tried to resolve this in commit ca5074f. Please review.

**Comment by kotp** [comment]
Apparently it is Sunday evening... missed doing this with you all.

**Comment by kotp** [comment]
Oh, man... missed reviewing this by 3 minutes.

**Pull request merged**